dashboard/src2/pages/Billing.vue

```diff
@@ -5,7 +5,20 @@
 				:items="[{ label: 'Billing', route: { name: 'Billing' } }]"
 			/>
 		</Header>
-		<TabsWithRouter :tabs="tabs" />
+		<TabsWithRouter
+			v-if="
+				$team.doc.is_desk_user ||
+				($session.roles.data || []).some(role => role.enable_billing)
```

Comment:
do this computation once in session.js, something like `$session.hasBillingAccess`